pulp_plugin_template/tests/functional/constants.py

```diff
@@ -8,6 +8,10 @@
     CONTENT_PATH
 )
 
+# FIXME: remove any download policies not supported by your plugin type
+DOWNLOAD_POLICIES = ['streamed', 'immediate', 'on_demand']
```

Comment:
since we don't support streamed or on_demand out of the box, maybe comment out this line and add a new line below:

    DOWNLOAD_POLICIES = ['immediate']

Reply:
I would use a `tuple`

```
DOWNLOAD_POLICIES = ('immediate',)
```